cfme/tests/configure/test_tag.py

```diff
@@ -107,6 +112,52 @@ def test_map_tagging_crud(appliance, category, soft_assert):
                                       .format(map_tag_entity.label))
 
 
+def test_tag_change_tag_name(provider, tag, request):
```

Comment:
Can you change the test name to  "test_updated_tag_name_on_vm"

Reply:
@sshveta ok, makes sense